arangod/RocksDBEngine/RocksDBRecoveryManager.cpp

```diff
@@ -182,22 +182,23 @@ class WBReader final : public rocksdb::WriteBatch::Handler {
       for (auto& pair : _deltas) {
         WBReader::Operations const& ops = pair.second;
         // now adjust the counter in collections which are already loaded
-        auto dbColPair = rocksutils::mapObjectToCollection(pair.first);
-        if (dbColPair.second == 0 && dbColPair.first == 0) {
+        RocksDBEngine::IndexTriple triple = rocksutils::mapObjectToIndex(pair.first);
+        if (std::get<0>(triple) == 0 && std::get<1>(triple) == 0) {
```

Comment:
it would be more readable to change the IndexTripple into a POD struct with 3 named members